internal/utils_test.go

```diff
@@ -266,3 +279,97 @@ func (token *mockedToken) Done() <-chan struct{} {
 func (token *mockedToken) Error() error {
 	return token.err
 }
+
+func createLocalArtifact(t *testing.T, dir, alias, body string) (file string, hash string) {
+	t.Helper()
+
+	file = filepath.Join(dir, alias)
+	if err := storage.WriteLn(file, body); err != nil {
+		t.Fatal("error writing to file", err)
+	}
+	hType := sha256.New()
+	hType.Write([]byte(body))
+	hash = hex.EncodeToString(hType.Sum(nil))
+	return
```

Comment:
It's my personal preference but I really think that this implicit return is not a good idea. The code is not so readable. I would suggest to have explicit return parameters.